src/openms/include/OpenMS/KERNEL/MRMTransitionGroup.h

```diff
@@ -335,6 +335,12 @@ namespace OpenMS
         }
       }
 
+      for (typename std::vector<SpectrumType>::iterator pr_it = precursor_chromatograms_.begin(); pr_it != precursor_chromatograms_.end(); ++pr_it)
```

Comment:
hmm basically you are controlling with tr_ids whether to take certain subgroups of transitions or not. Would it make sense to have a second parameter that controls whether to add the precursor (and if so, which precursors?)